pandas/_libs/tslibs/parsing.pyx

```diff
@@ -745,6 +748,38 @@ cdef datetime dateutil_parse(
     return ret
 
 
+cdef object _reso_pattern = re.compile(r"\d:\d{2}:\d{2}\.(?P<frac>\d+)")
```

Comment:
Are the first `\d`s always guaranteed to be separate by `:`?

Reply:
dateutil supports some really weird formats (@MarcoGorelli  and i have discussed moving away from using it at all) so i dont know.  but i think this covers the vast majority of cases we care about